frontend/src/container/Header/DateTimeSelection/config.ts

```diff
@@ -46,7 +46,7 @@ export const getDefaultOption = (route: string): Time => {
 	if (route === ROUTES.SERVICE_MAP) {
 		return ServiceMapOptions[0].value;
 	}
-	if (route === ROUTES.APPLICATION) {
+	if (route === ROUTES.APPLICATION || '/') {
```

Comment:
please don't use this one.. this is not the right solution

Reply:
OK, but there is no problem if we remove it still, the application works perfectly because of localStorage.